cmd/kubectl-direct_csi/volumes_list.go

```diff
@@ -51,6 +56,12 @@ $ kubectl direct-csi volumes ls --nodes=directcsi-1
 
 # Combine multiple filters using csv
 $ kubectl direct-csi vol ls --nodes=directcsi-1,directcsi-2 --status=staged --drives=/dev/nvme0n1
+
+# List all volumes of pod name
+$ kubectl direct-csi volumes ls --pod-name=my-minio*
```

Comment:
The `pod.name` and `pod.namespace` labels will be available only on "published" volumes. Can we explain that in the example here. Like, 

```
# List volumes published for the provided pods
$ kubectl direct-csi volumes ls --pod-name=my-minio* --status=published

# List volumes published in the provided namespaces
kubectl direct-csi volumes ls --pod-namespace=my-minio-ns* --status=published
```

Reply:
Done